workers/ohsome_quality_analyst/oqt.py

```diff
@@ -57,8 +57,12 @@ async def _(
     Returns:
         Depending on the input a single indicator as GeoJSON Feature will be returned
         or multiple indicators as GeoJSON FeatureCollection will be returned.
+
+    Note:
+        Indicators for a FeaturesCollection are created asynchronously defaulting to
+        four executions at a time (semaphore).
```

Comment:
Is it good to mention the default value here, since it is defined in `sem_var(…)`?

Reply:
Since I plan to make this value configurable (#255) mentioning of the default value is indeed not necessary here. I removed the mentioning of the default value in every docstring. Also moved the 'Note' section to the main part of the docstring.